ext/drampower/src/CAHelpers.cc

```diff
@@ -75,18 +75,19 @@ int64_t CommandAnalysis::timeToCompletion(MemCommand::cmds type)
 // Returns the number of active banks based on the bank_state vector.
 unsigned CommandAnalysis::get_num_active_banks(void)
 {
-  return (unsigned)std::count(bank_state.begin(), bank_state.end(), BANK_ACTIVE);
+  return bank_state.get_num_active_banks();
+  //return (unsigned)std::count(bank_state.begin(), bank_state.end(), BANK_ACTIVE);
```

Comment:
Please remove lines that should be deleted instead of commenting them. If there is some important history either (a) we can go back and look at it in the git history or (b) add a prose comment about why the history is important.